datasets/doc2dial/README.md

```diff
@@ -68,11 +70,322 @@ English
+    "title": "Benefits Planner: Retirement | Online Calculator (WEP Version)#1_0",
+    "domain": "ssa"
+}
+```
+
+
+
+
+
+
 
 ### Data Fields
 
-[More Information Needed]
+For `document_domain`,
+
+- `doc_id`: the ID of a document;
+- `title`: the title of the document;
+- `domain`: the domain of the document;
+- `doc_text`: the text content of the document (without HTML markups);
+- `doc_html_ts`: the document content with HTML markups and the annotated spans that are indicated by `text_id` attribute, which corresponds to `id_sp`.
+- `doc_html_raw`: the document content with HTML markups and without span annotations.
+- `spans`: key-value pairs of all spans in the document, with `id_sp` as key. Each span includes the following,
+  - `id_sp`: the id of a  span as noted by `text_id` in  `doc_html_ts`;
+  - `start_sp`/  `end_sp`: the start/end position of the text span in `doc_text`;
+  - `text_sp`: the text content of the span.
+  - `id_sec`: the id of the (sub)section (e.g. `<p>`) or title (`<h2>`) that contains the span.
+  - `start_sec` / `end_sec`: the start/end position of the (sub)section in `doc_text`.
+  - `text_sec`: the text of the (sub)section.
+  - `title`: the title of the (sub)section.
+  - `parent_titles`: the parent titles of the `title`.
+
+
+
+For `dialogue_domain`:
+
+- `dial_id`: the ID of a dialogue;
+- `doc_id`: the ID of the associated document;
+- `domain`: domain of the document;
+- `turns`: a list of dialogue turns. Each turn includes,
+  - `turn_id`: the time order of the turn;
+  - `role`: either "agent" or "user";
+  - `da`: dialogue act;
+  - `references`: the grounding span (`id_sp`) in the associated document. If a turn is an irrelevant turn, i.e., `da` ends with "ood", `reference` is empty.  **Note** that spans with labels "*precondition*"/"*solution*" are the actual grounding spans. Spans with label "*reference*" are the related titles or contextual reference,  which is used for the purpose of describing a dialogue scene better to crowd contributors.
+  - `utterance`: the human-generated utterance based on the dialogue scene.
+
+
+
+For `doc2dial_rc`,
```

Comment:
This looks really convenient to have the reading comprehension format !
Can you add more details about this format (advantages, how it is generally used) ? Or maybe redirect to a webpage that gives more details ?

Reply:
Thanks! I have updated README with some more information.

Reply:
Thanks :)
Feel free to also resolve the conflicts of this branch

Reply:
I have tried to merge `master` to `doc2dial_rc` branch, but I don't have the write access and the conflicts are still there. To resolve conflicts, I'd accept the `doc2dial_rc`. Could you please help understand how I shall proceed? Thank you!

Reply:
What's the error you're getting ?
Did you try merging this way ?
```
git checkout doc2dial_rc
git fetch upstream
git merge upstream master
```

Let me know if this works. Otherwise I can take care of the merge

(also if you didn't have the base repository as a remote already you can do
```
git remote add upstream https://github.com/huggingface/datasets.git
```
)

Reply:
Thanks so much! The issue is that I don't get any error. Just tried that again but as I have done it before so there is `nothing to commit and working tree clean`. I do have the remote upstream already... But the conflicts seem still there... Sorry about the inconvenience!